packages/components/nodes/chains/ApiChain/OpenAPIChain.ts

```diff
@@ -0,0 +1,84 @@
+import { ICommonObject, INode, INodeData, INodeParams } from '../../../src/Interface'
+import { APIChain, createOpenAPIChain } from 'langchain/chains'
+import { CustomChainHandler, getBaseClasses } from '../../../src/utils'
+import { ChatOpenAI } from 'langchain/chat_models/openai'
+
+class OpenApiChain_Chains implements INode {
+    label: string
+    name: string
+    type: string
+    icon: string
+    category: string
+    baseClasses: string[]
+    description: string
+    inputs: INodeParams[]
+
+    constructor() {
+        this.label = 'OpenAPI Chain'
+        this.name = 'openApiChain'
+        this.type = 'openApiChain'
+        this.icon = 'apichain.svg'
```

Comment:
can we use the image [here](https://github.com/FlowiseAI/Flowise/blob/main/packages/components/nodes/tools/OpenAPIToolkit/openapi.png)

Reply:
Sure will make the change forgot about this.

Reply:
<img width="1431" alt="Screenshot 2023-07-07 at 6 35 32 PM" src="https://github.com/FlowiseAI/Flowise/assets/405556/f8417bd4-70b0-4dfc-bf34-6a37922bd870">